contrib/intel/jenkins/Jenkinsfile

```diff
@@ -294,6 +295,338 @@ pipeline {
                   echo "MPI-tcp-rxm-1 completed."
                 )
               """
+=======
```

Comment:
It doesn't look like you resolved this confict.
You need to keep the current upstream jenkinsfile and just add your multinode-test stage.
Make sure to keep the spacing correct. It is using two spaces as indentation and not tabs.